imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -175,7 +106,12 @@ def get_histogram(
     hist : np.ndarray
         A 3D histogram array.
     """
-    az, el, _ = cartesian_to_spherical(v)
+    spherical_coords = cartesian_to_spherical(v)
+    az, el, _ = (
+        spherical_coords[..., 0],
+        spherical_coords[..., 1],
+        spherical_coords[..., 2],
+    )
```

Comment:
No need to include the radius in the right hand side:
```suggestion
    az, el = (
        spherical_coords[..., 0],
        spherical_coords[..., 1],
    )
```